packages/superset-ui-connection/src/SupersetClientClass.ts

```diff
@@ -70,12 +70,17 @@ export class SupersetClientClass {
     return this.csrfToken !== null && this.csrfToken !== undefined;
   }
 
-  async get({
+  async get(requestConfig: RequestConfig): Promise<SupersetClientResponse> {
```

Comment:
if you're refactoring `get` like this, I think you should also refactor `post`, and then could add `delete` / `put` on top of those.

Reply:
+1